tests/cli/test_init.py

```diff
@@ -19,38 +19,56 @@
 from pathlib import Path
 
 import pytest
-from tests.core.commands.test_init import TEMPLATE_NAME, TEMPLATE_REF, \
-    TEMPLATE_URL
+from tests.core.commands.test_init import TEMPLATE_ID, TEMPLATE_INDEX, \
+    TEMPLATE_REF, TEMPLATE_URL
 
 from renku.cli import cli
 from renku.cli.init import create_template_sentence
 
-INIT = ['init', 'test-new-project', '--template', TEMPLATE_NAME]
+INIT = ['init', 'test-new-project', '--template-id', TEMPLATE_ID]
 INIT_REMOTE = [
     '--template-source', TEMPLATE_URL, '--template-ref', TEMPLATE_REF
 ]
 INIT_FORCE = ['--force']
 INIT_VARIABLES = ['--template-variables']
+INIT_INDEX = ['init', 'test-new-project-2', '--template-index', TEMPLATE_INDEX]
+INIT_ID = ['--template-id', TEMPLATE_ID]
+LIST_TEMPLATES = ['--list-templates']
 
 
 def test_template_selection_helpers():
     templates = [{
         'name': 'Template Python',
+        'folder': 'folder_python',
         'description': 'Description Python'
     }, {
         'name': 'Template R',
+        'folder': 'folder_R',
         'description': 'Description R'
     }]
-    instructions = 'Please choose a template by typing the number'
+    instructions = 'Please choose a template by typing the index'
     sentence = create_template_sentence(templates)
     stripped_sentence = ' '.join(sentence.split())
-    assert '1 Template Python Description Python' in stripped_sentence
-    assert '2 Template R Description R' in stripped_sentence
+    assert (
+        '1 folder_python Template Python: Description Python'
+    ) in stripped_sentence
+    assert '2 folder_R Template R: Description R' in stripped_sentence
     assert instructions not in stripped_sentence
     full_sentence = create_template_sentence(templates, True)
     assert instructions in full_sentence
 
 
+@pytest.mark.integration
```

Comment:
This test does not need to be an integration test. We normally use this marker when the test needs to access the Internet.

Reply:
ok good to know, then I should remove it also from other tests here, only 1 is accessing internet to download a repo